documentation/api/query/curl.markdown

```diff
@@ -81,3 +81,14 @@ If you do this with an endpoint that accepts `GET` requests, **you must also use
 
     curl -G http://localhost:8080/pdb/query/v4/nodes \
       --data-urlencode 'query=["=", ["node", "active"], true]'
+
+## Querying PuppetDB with POST
+
+PuppetDB supports querying by POST, which is useful for particularly large
+queries (exact limits depend on the client and webserver used.)
+
+POST queries use the following syntax:
+
+    curl -X POST http://localhost:8080/pdb/query/v4/nodes \
+    -H 'Content-Type:application/json' \
```

Comment:
Indent the trailing lines 2 space, seems to be a convention we follow in the rest of the examples (also mildly easier to read).